components/fxa-client/examples/devices_api.rs

```diff
@@ -94,9 +94,9 @@ fn main() -> Result<(), failure::Error> {
                 persist_fxa_state(&acct.lock().unwrap());
                 for e in evts {
                     match e {
-                        AccountEvent::TabReceived((device, payload)) => {
+                        IncomingDeviceCommand::SendTab { sender, payload } => {
```

Comment:
The Kotlin wrapped appears to call this `IncomingDeviceCommand::TabReceived`, while here it's called `IncomingDeviceCommand::SendTab`. I think it would be clearer if we can make them both consistent (yes, I am imagining a world where one is automatically generated from the other...) and I lightly prefer `TabReceived` since it's phrased from the perspective of the receiving device.

Reply:
(It's also fine to keep them differently named if there's a reason for it, but it wasn't obvious to me why they should be different.)